src/breakpoints/index.ts

```diff
@@ -52,13 +67,76 @@ class BreakpointsHeader extends Widget {
 }
 
 export namespace Breakpoints {
+  export interface IBreakpoint extends DebugProtocol.Breakpoint {}
+
   /**
    * The breakpoints UI model.
    */
   export interface IModel {}
 
+  export class IModel implements IModel {
+    constructor(model: IBreakpoint[]) {
+      this._state = model;
+    }
+
+    get breakpointsChanged(): ISignal<this, IBreakpoint[]> {
+      return this._breakpointsChanged;
+    }
+
+    get activeChanged(): ISignal<this, boolean> {
+      return this._activeChange;
+    }
+
+    get breakpoints(): IBreakpoint[] {
+      return this._state;
+    }
+
+    get isActive(): boolean {
```

Comment:
We will probably want to have control on the active state at the breakpoint level rather than all the breakpoints at once (can be done in follow-up PRs)